src/test/java/de/cyface/collector/FileUploadTest.java

```diff
@@ -317,7 +317,11 @@ private void upload(final TestContext context, final String testFileResourceName
         final URL testFileResource = this.getClass().getResource(testFileResourceName);
 
         form.binaryFileUpload("fileToUpload",
-                String.format(Locale.US, "%s_%s.cyf", deviceIdentifier, measurementIdentifier),
+                String.format(Locale.US, "%s_%s.ccyf", deviceIdentifier, measurementIdentifier),
```

Comment:
Siehe in der openapi.yml - da steht ".ccyf" ist die Dateiendung für den compressed upload.
Daher habe ich das im SDK auch so geändert. Oder ist das Problematisch?
MOV betrifft das nicht, da die Änderungen nur in den dev gehen und somit erst im V5 Release inkl. sind.